### ISSUE TITLE
List-ID headers should conform to RFC https://tools.ietf.org/html/rfc2919

### ISSUE THREAD

**Issue by mrvdb** [odoo/odoo#18408]
Impacted versions:
Git version: see https://github.com/odoo/odoo/blob/10.0/addons/mail/models/mail_channel.py#L200

Steps to reproduce:
 Receive messages from ODOO mailinglists and view message source

Current behavior:
List-Id headers have the form, for example, 

  `List-Id: community.mail.odoo.com`

Expected behavior:
List-Id headers should have the form:

`"List-ID:" [phrase] "<" list-id ">" CRLF`

In short, there should be a list-id between pointy brackets optionally preceded by a phrase.

**Comment by Yenthe666** [comment]
@odony I know you have a lot of experience with e-mails and e-mail headers so could you please take a look and give feedback?

**Comment by odony** [comment]
Thanks for reporting, you're right indeed. This is fixed by 962af4b7807090af9a28cca295fb3b9b98a4cea6 and will be forward-ported in the coming days to later versions.
